b/binaryextensions/binaryextensions_ubi_8.3.sh

```diff
@@ -0,0 +1,88 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : binaryextensions
+# Version               : 2.2.0
+# Source repo           : https://github.com/bevry/binaryextensions
+# Tested on             : UBI 8.3
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Arumugam N S <asellappen@yahoo.com> / Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+
+export REPO=https://github.com/bevry/binaryextensions
+
+#clone from default tag v2.2.0 
+if [ -z "$1" ]; then
+  export VERSION="v2.2.0"
+else
+  export VERSION="$1"
+fi
+
+if [ -d "binaryextensions" ] ; then
+  rm -rf binaryextensions
+fi
+
+#Install Dependencies
+if [ ! -d ~/.nvm ]; then
+    sudo yum update -y
+    sudo yum groupinstall 'Development Tools' -y
+    sudo yum install -y openssl-devel.ppc64le curl git
+    sudo curl https://raw.githubusercontent.com/creationix/nvm/v0.37.2/install.sh | bash
```

Comment:
were we not able to simply use "dnf module install nodejs:12" for installing nodejs. Rest all LGTM

Reply:
both should be fine , Earlier i have used dnf only and worked fine. But i have seen Siddhesh released at as above at  https://github.com/ppc64le/build-scripts/pull/969/files  in one of his release recently ,So i thought of following the one which got merged .